Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -184,7 +184,7 @@
     xpc_dictionary_set_string(bootstrapMessage.get(), "service-name", name);
 
     if (m_launchOptions.processType == ProcessLauncher::ProcessType::Web) {
-        bool disableLogging = true;
+        bool disableLogging = false;
```

Comment:
😅

Reply:
(Indeed, I fixed this while I was fixing the failing API test — `NetworkProcess.LaunchOnlyWhenNecessary`)